src/arm/BBORG_SERVO-00A2.dts

```diff
@@ -35,27 +35,27 @@
 
```

Comment:
Can you add some comments above about the overall function and resources. I think I know now, but a summary in the comments would be nice.

Reply:
Sure thing :+1: